### DISCUSSION TITLE
本地webui的API接入

### DISCUSSION THREAD

**Discussion by panda114514667** [STranslate/STranslate#127] [Q&A]
我想问一下，我使用LM studio的自建api和koboldcpp的服务，得到的本地api接口在填入软件翻译服务-自建翻译里，点验证显示失败。尝试翻译也是请求失败。是我的api设置问题吗？有什么解决办法。

**Comment by ZGGSONG** [comment]
你所使用的服务api接口内容是啥呢，这个很重要，我没有使用过。如果提供我可以帮你分析一下。

> 自建服务所支持的api在填入信息页面有服务支持的api要求，请检查是否符合

**Reply by panda114514667** [reply]
一般来说启动webui后会自动生成一个本地API大概是{http://localhost:端口号/api/}在本地或局域网运行的AI应用都可以用这个接口。启动平台是Koboldcpp在git上也有项目。

**Comment by ZGGSONG** [comment]
![image](https://github.com/user-attachments/assets/69a9306a-7ccb-4c8e-be94-d25d4f5deff6)

api 你大概率还是post的api  body是啥   是否是流式传输

**Comment by panda114514667** [comment]
![koboldcpp_nocuda_05S6sKbZrG](https://github.com/user-attachments/assets/f3bc816c-5e55-44f1-a31e-65dcfd318711)
![LM_Studio_VLss1vEHIN](https://github.com/user-attachments/assets/a2e46a2f-95d9-44b6-afab-191be2ae0e6c)
我不知道哪边你看的方便点，LMstudio在有个支持文档https://lmstudio.ai/docs/local-server

**Comment by ZGGSONG** [comment]
![image](https://github.com/user-attachments/assets/c23e6317-5fb6-4fb8-9ef2-489c7c1cf5a9)
![image](https://github.com/user-attachments/assets/1672bfc8-7e4b-425d-80ee-f25780901acc)
你直接使用OpenAI的服务类型即可
![image](https://github.com/user-attachments/assets/cd4a5348-f1fa-44ec-8506-23e5775df93c)

**Comment by panda114514667** [comment]
十分感谢，webui已经成功接入，你的项目常帮助到我。是好的项目。(^y^)
